### PR TITLE
Added Redux State

### PR DESCRIPTION
**Pull request by solomojb**

### CODE REVIEW

**Comment by heisch** [comment]
I'll have a real look at this this week and will do some testing, but looking good on the first glance 👍

**Comment by solomojb** [comment]
Great thanks.

I have a second set of changes regarding a react refactor all ready to follow once that’s merged. :)

I have more planned but I’ll hold off as this second change is big. :)

If you prefer I merge that and do it as one giant pr let me know so you don’t have to test twice.

Jim

Sent from my iPhone

On Jul 7, 2020, at 3:01 AM, Nicola Heisch <notifications@github.com> wrote:

﻿

I'll have a real look at this this week and will do some testing, but looking good on the first glance 👍

File: src/State/ItemViewState.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+export function storeFilterSlot(slot?: GloomhavenItemSlot) {
+    return { type: STORE_FILTER_SLOT, slot}
+}
+
+export function storeFilterSearch(search: string) {
+    return { type: STORE_FILTER_SEARCH, search}
+}
+
+export function storeSortingProperty(property: SortProperty) {
+    return { type: STORE_FILTER_SLOT, property}
```
Comment (solomojb) [inline src/State/ItemViewState.tsx:23]: This should be STORE_SORTING_PROPERTY. I have fixed this in another branch

**Comment by heisch** [comment]
Merged and deployed 🎉

**Pull request merged**